app/operations/ops/developer/remove_role.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Ops
+  module Developer
+    # Removes the role from user
+    class RemoveRole < BaseOperation
+      step :remove_role!
```

Comment:
You can even embed the exception class inside this class.
It'll be the most simple way.